blackjax/mcmc/integrators.py

```diff
@@ -1,3 +1,4 @@
+# @title `integrators.py` from https://github.com/blackjax-devs/blackjax/pull/589
```

Comment:
revert

Reply:
Can you explain?

Reply:
I mean remove this change (since it was added because you were copying the changes from the integrator PR)